src/index.spec.ts

```diff
@@ -523,11 +551,14 @@ describe('ts-node', function () {
   })
 
   describe('register', function () {
-    const registered = register({
-      project: PROJECT,
-      compilerOptions: {
-        jsx: 'preserve'
-      }
+    let registered: tsNodeTypes.Register
+    before(() => {
+      registered = register({
+        project: PROJECT,
+        compilerOptions: {
+          jsx: 'preserve'
+        }
+      })
```

Comment:
Since ts-node is now being loaded inside a `before()` callback, we must also register ts-node inside a `before()` callback.